app/views/partials/_header.html.haml

```diff
@@ -23,7 +23,7 @@
                             %li.buttoned
                                 = link_to 'Log out', destroy_user_session_path, method: :delete
                         - else 
-                            %li
+                            %li.buttoned
```

Comment:
Why do you use buttonend on both li elements.  Shouldn’t only the last one have that class?

Reply:
If .buttoned is not applied then `Log in` will not have the yellow border on it.

Reply:
Oh I see now, that `Log in` is not supposed to have a yellow border according to the layout-pdf